.github/workflows/ci.yml

```diff
@@ -132,7 +132,8 @@ jobs:
       - run: go install github.com/cpuguy83/go-md2man/v2@v2.0.2
       - run: make man
 
-  # Make sure binaries compile with other platforms
+  # Make sure binaries compile with other minor platforms.
+  # Well-known architectures are covered in release.yml.
```

Comment:
Question on removing some cross platforms builds.  They are covered in release.yml, however release.yml will only be executed on release, i.e. tags matching v*.   
Shouldn't the check be executed on all PRs into main to catch issues early?

Reply:
Ok, see the second commit updated release.yml to run on PRs too. 

Looks good to me.